cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/documentstore/Attachment.java

```diff
@@ -128,9 +112,6 @@ public boolean equals(Object o) {
 
         Attachment that = (Attachment) o;
 
-        if (name != null ? !name.equals(that.name) : that.name != null) {
-            return false;
-        }
```

Comment:
I've gone for 4 in 667bdaf - we don't need to compare `Attachment`s for equality.